air/src/execution_step/resolver/resolvable_impl.rs

```diff
@@ -71,9 +73,17 @@ impl Resolvable for Option<LambdaAST<'_>> {
             provenance,
         } = ctx.last_error();
 
-        let jvalue = match self {
-            Some(error_accessor) => select_by_lambda_from_scalar(error.as_ref(), error_accessor, ctx)?.into_owned(),
-            None => error.as_ref().clone(),
+        let jvalue = if !error.is_null() {
+            match self {
+                Some(error_accessor) => select_by_lambda_from_scalar(error.as_ref(), error_accessor, ctx)?.into_owned(),
+                None => error.as_ref().clone(),
+            }
+        } else {
+            let error = json!({"error_code": NO_ERROR_ERROR_CODE, "message": NO_ERROR_MESSAGE});
```

Comment:
There are `ERROR_CODE_FIELD_NAME` and family constants. There is also `error_from_raw_fields` function, but it adds more fields (and it may be ok).